backend/controllers/faq.js

```diff
@@ -0,0 +1,70 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const cors = require('cors')
+const faqRouter = express.Router()
+
+var mongodb = require('mongodb');
+var Faq = require('../models/FAQ');
+
+faqRouter.use(cors())
```

Comment:
Do we need to do these to all the Routers? Isn't there a way to just do it once? Do we need a Router for every single page? Can't we just use one instantiation of the router object?

Reply:
we can but google says it's good practice to split up routers based on subjects lol i think it's probably better to decouple things not related to each other anyway

Reply:
Ok. I can live with that.